standards/request-response.md

```diff
@@ -493,6 +493,36 @@ The usage of non-standard headers is not considered custom headers. For example,
 SPS-CORS-Error: bad origin
 ```
 
+#### SPS-Execution-Context
+
+**Type**: Request
+
+**Support**: OPTIONAL
+
+**Description**: The Execution Context header provides a standard method of indicating the dataflow context that a request should be processed with. This is used to differentiate or test changes in customer configuration of non-production data. This isolates the version of the configuration state used in a request and allows identifying transactions that should leverage a specific tagged or aliased configuration set.
+
+- A request without the presence of this header **MUST** indicate that the execution context is under the `production` dataflow and customer configuration, if applicable.
```

Comment:
I couldn't help but think that, this concept really doesn't apply to EVERY API ever made?

Reply:
ah, in that context. No, some APIs don't need it. Maybe we should separate the fact of APIs that by concept support it but don't have this header and APIs that are not part of transactional processing (or don't have to support context) and that's where it will be non-applicable case.

Reply:
I maybe read too deep into it - "if applicable" in that sentence corresponds to the behavior of using "prod" when no header is provided as it reads. There is no API mentioned in the sentence itself, and people might try to apply "if applicable" to described behavior and not to the fact that API does not support context by design. Maybe just clarifying like "if applicable to this API" will be sufficient.

Reply:
ah gotcha! I understand now! Make sense.